a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -8,6 +8,7 @@
 		<script class="remove">
 			// <![CDATA[
 			var respecConfig = {
+				shortName: "epub-techniques",
```

Comment:
Shouldn't this short name be more explicit?
Like with a `a11y-display-` prefix?

Reply:
This is only for the dated final version that's published on the w3c server. For example:
https://www.w3.org/community/reports/publishingcg/CG-FINAL-onix-techniques-20250422/

It doesn't need to be too specific as this isn't how people will commonly access the guide and techniques.

The current version links that we'll use for referencing are already clear enough:
https://www.w3.org/publishing/a11y/metadata-display-guide/techniques/onix/